--- patch27.c
+++ patch28.c
@@ -745,11 +745,11 @@
 MSMDiffResult *msm_diff(const char *a_str, const char *b_str)
 {
     int new_a_len_420186 = strlen(a_str);
-    int b_len = strlen(b_str);
+    int new_b_len_269522 = strlen(b_str);
 
     int raw_diff_count = 0;
     MSMRawDiff *raw_diffs =
-        msm_diff_chars(a_str, new_a_len_420186, b_str, b_len, &raw_diff_count);
+        msm_diff_chars(a_str, new_a_len_420186, b_str, new_b_len_269522, &raw_diff_count);
 
     if (!raw_diffs)
         return NULL;
